### DISCUSSION TITLE
Code review IM-12

### DISCUSSION THREAD

**Discussion by MytsV** [HowProgrammingWorks/LiveQA#57] [Code review]
2 семестр, курсова робота

Я займаюся розробкою **браузерного застосунку для створення й поширення pixel-art анімацій**.
Основа проекту схожа на класичний Paint часів Windows XP з розширеними можливостями як-от робота зі слоями, кадрами та експорт в GIF формат.
У проекті не використовуються ніякі залежності окрім влаштованого API браузера.
У майбутньому я планую доповнити браузерний застосунок роботою з "serverless" backend-ом (скоріш за все, платформою Firebase) використовуючи лише засоби влаштованого Fetch API.
Короткий список з основними запланованими features додатку разом з кодом можна знайти в [репозиторії проекту](https://github.com/MytsV/pixel-toons-editor/tree/dev).

Буду дуже вдячна за рев'ю проекту!

**Comment by tshemsedinov** [comment]
https://github.com/MytsV/pixel-toons-editor/issues/15
